### PR TITLE
Mention install.sh in README.md

### PR DESCRIPTION
**Pull request by yeonghoey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ There's a few special files in the hierarchy.
   expected to setup `$PATH` or similar.
 - **topic/completion.zsh**: Any file named `completion.zsh` is loaded
   last and is expected to setup autocomplete.
+- **topic/install.sh**: Any files named `install.sh` is executed when you run `script/install`.
```
Comment (jacobwgillespie) [inline README.md]: This should read "Any file named"

**Comment by yeonghoey** [comment]
I applied your feedback. Fixed the same problem in the line below.
Add more explanation which makes sense I think.

**Comment by DamnedScholar** [comment]
Line 44 should read, "Any file ending in `*.symlink` _gets_ symlinked".

**Comment by StewartJarod** [comment]
there weren't any modifications to that line... should be good to go

**Comment by yeonghoey** [comment]
I missed the notification. Applied the feedback (**gets** symlinked)

**Comment by holman** [comment]
Keep forgetting this isn't in there already. Thanks!

**Pull request merged**